### PR TITLE
Identity 1071

### PR DESCRIPTION
**Pull request by wattale**

### CODE REVIEW

File: components/oauth/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/cache/SessionDataCache.java [hunk 1]
```diff
@@ -18,37 +18,64 @@
...
 // Cache used by Authorization endpoint. This class cannot be in oauth.endpoint component
 // since it needs to be visible to Hazelcast.
-public class SessionDataCache extends BaseCache<CacheKey, CacheEntry> {
+public class SessionDataCache extends BaseCache<String, CacheEntry> {
 
     private static final String SESSION_DATA_CACHE_NAME = "OAuthSessionDataCache";
 
-    private static final SessionDataCache instance = new SessionDataCache(SESSION_DATA_CACHE_NAME);
+    private static volatile SessionDataCache instance;
+	private boolean enableRequestScopeCache = false;
```
Comment (hpmtissera) [inline components/oauth/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/cache/SessionDataCache.java]: Format the code in all changed lines

File: components/oauth/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/config/OAuthServerConfiguration.java [hunk 2]
```diff
@@ -77,6 +77,11 @@
     private long applicationAccessTokenValidityPeriodInSeconds = 3600;
     private long refreshTokenValidityPeriodInSeconds = 24 * 3600;
     private long timeStampSkewInSeconds = 300;
+	private int sessionDataCacheTimeout = -1;
```
Comment (hpmtissera) [inline components/oauth/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/config/OAuthServerConfiguration.java]: Format the code properly

File: components/sts/org.wso2.carbon.identity.sts.passive.ui/pom.xml [hunk 3]
```diff
@@ -70,6 +70,7 @@
                             org.wso2.carbon.identity.application.authentication.common.*,
                             javax.servlet;version="${imp.pkg.version.javax.servlet}",
                             javax.servlet.http;version="${imp.pkg.version.javax.servlet}",
+                            org.wso2.carbon.identity.application.authentication.framework.*,
```
Comment (hpmtissera) [inline components/sts/org.wso2.carbon.identity.sts.passive.ui/pom.xml]: need to set the import version correctly

**Comment by wattale** [comment]
Closing this due to merge conflicts. I will send another PR for this

**Pull request closed without merging**